packages/firebase_auth/ios/Classes/FirebaseAuthPlugin.m

```diff
@@ -68,6 +69,29 @@ - (FIRAuth *_Nullable)getAuth:(NSDictionary *)args {
   return [FIRAuth authWithApp:[FIRApp appNamed:appName]];
 }
 
+- (bool)application:(UIApplication *)application
+    didReceiveRemoteNotification:(NSDictionary *)notification
+          fetchCompletionHandler:(void (^)(UIBackgroundFetchResult result))completionHandler {
+  if ([[FIRAuth auth] canHandleNotification:notification]) {
+    completionHandler(UIBackgroundFetchResultNoData);
```

Comment:
if completionHandler is nullable, `completionHandler(UIBackgroundFetchResultNoData);` might crash if completionHandler is null. 
Ignore this comment if completionHandler is nonnull.

Reply:
This code sample comes straight from Firebase so I think it's probably safe.

Reply:
https://firebase.google.com/docs/auth/ios/phone-auth